src/libraries/System.Private.Uri/src/System/UriScheme.cs

```diff
@@ -159,7 +159,7 @@ public static void Register(UriParser uriParser, string schemeName, int defaultP
             if ((defaultPort >= 0xFFFF || defaultPort < 0) && defaultPort != -1)
                 throw new ArgumentOutOfRangeException(nameof(defaultPort));
 
-            schemeName = schemeName.ToLower();
+            schemeName = schemeName.ToLowerInvariant();
```

Comment:
This doesn't need to be culture-aware?

Reply:
I don't think so, as I believe these are intended to represent tokens / identifiers that shouldn't be treated in a linguistic-aware fashion. @scalablecory, would you be able to confirm?

Reply:
According to [RFC3986](https://tools.ietf.org/html/rfc3986), the scheme part of an URI has the following grammar:

      scheme      = ALPHA *( ALPHA / DIGIT / "+" / "-" / "." )

Moreover, it says this about the kind of characters that are considered part of URIs:

                             A URI is a sequence of characters from a
      very limited set: the letters of the basic Latin alphabet, digits,
      and a few special characters. 

So I think it's safe to say that being culture invariant here is the way to go.